addons/captives/stringtable.xml

```diff
@@ -247,6 +247,7 @@
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_name">
             <English>Require surrendering</English>
+            <German>Benötigt Kapitulation</German>
```

Comment:
imho, "Kapitulation"  is not used for individual soldiers - as opposed to "sich ergeben/Ergebung"